scripts/background.js

```diff
@@ -49,7 +49,7 @@ class BackgroundScript {
     let taskName = card.name.match(/(?<=#)[A-Za-z0-9_-]+/)?.[0];
 
     if (taskName === undefined) {
-      taskName = `${taskPrefix}_${card.idShort}`;
+      taskName = `${taskPrefix}_${card.idShort}_${card.idBoard.substring(0, 3)}`;
```

Comment:
@aboutnick @wendelin-as Also ich finde diese Lösung eigentlich besser. Was spricht denn dagegen? Warum habt ihr euch für diese Lösung entschieden?

```suggestion
      taskName = `${taskPrefix}_${card.shortLink}`;
```

Reply:
wie besprochen wird jetzt die card id plus shortLink verwendet.